src/System.Net.Http.WinHttpHandler/tests/FunctionalTests/System.Net.Http.WinHttpHandler.Functional.Tests.csproj

```diff
@@ -31,5 +31,8 @@
   <ItemGroup>
     <None Include="project.json" />
   </ItemGroup>
+  <ItemGroup>
+    <Compile Include="FakeDiagnosticListenerObserver.cs" />
+  </ItemGroup>
```

Comment:
Fixed (in the sense that all changes under :/src/system.net.http.winhttphandler/tests were reverted).